packages/web/utils/formatter.ts

```diff
@@ -392,3 +393,9 @@ export function calcFontSize(numChars: number, isMobile: boolean): string {
 
   return "16px";
 }
+
+export function trimmedPriceWithSymbol(price: PricePretty | undefined) {
+  return `${price?.symbol}${trimPlaceholderZeros(
+    price?.toDec().toString(2) ?? ""
+  )}`;
+}
```

Comment:
I prefer not to introduce another general formatting option since we already have formatPretty. This function, originally named formatSpendLimit, should keep its name because it is specifically used for displaying the spend limit. I don’t foresee any future use cases for an alternative. WDYT?

Reply:
makes sense

this was not a good place for this formatter

ty

Reply:
122440450425